iam_policy/variables.tf

```diff
@@ -1,13 +1,3 @@
-variable "profile" {
-  description = "AWS Profile"
-  default     = "default"
-}
-
-variable "region" {
-  description = "AWS Default Region"
-  default     = "us-east-1"
-}
-
```

Comment:
Since these are being removed, can they also be removed from the README.md?

Reply:
Done